### DISCUSSION TITLE
请求参数如何去掉必填验证

### DISCUSSION THREAD

**Discussion by fzwu** [luoyunchong/lin-cms-dotnetcore#33] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/16144784/226536097-b454e38e-bdb2-40ee-aa20-35dae8770439.png)
![image](https://user-images.githubusercontent.com/16144784/226536201-fcf2a672-7a98-40b1-b1c3-71b03600503c.png)

请求参数定义没有添加Required

**Comment by fzwu** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/16144784/226547885-2dbd44eb-148a-4e33-b169-3b4ab37f0573.png)
.NET 7验证非空及必填，定义实体增加？可为空即可

**Reply by luoyunchong** [reply]
https://github.com/luoyunchong/lin-cms-dotnetcore/blob/master/src/LinCms.Web/LinCms.Web.csproj
 这个是全局的	
`<Nullable>enable</Nullable>`

**Reply by fzwu** [reply]
已经配置<Nullable>enable</Nullable>，但是请求接口时，入参实体依旧需要设置？才可以，如果不增加？，则显示字段必须

**Reply by luoyunchong** [reply]
enable是启用，意思是开启全局 Null检查，string就要检查非NULL，不想用，设置成disable

**Reply by fzwu** [reply]
好的，多谢
